x-pack/plugins/lens/public/xy_visualization/expression.tsx

```diff
@@ -59,6 +58,8 @@ import { getAxesConfiguration, GroupsConfiguration, validateExtent } from './axe
 import { getColorAssignments } from './color_assignment';
 import { getXDomain, XyEndzones } from './x_domain';
 import { getLegendAction } from './get_legend_action';
+import { i18n } from '@kbn/i18n';
+import { I18nProvider } from '@kbn/i18n/target_types/react';
```

Comment:
Why have we changed this?

Reply:
this line is preventing `lens` plugin to compile on my machine:

> ERROR in ./public/xy_visualization/expression.tsx
Module not found: Error: Can't resolve '@kbn/i18n/target_types/react' in '/x-pack/plugins/lens/public/xy_visualization'

Reply:
Sorry Im not sure why that was changed - fixing now!